src/goMain.ts

```diff
@@ -118,15 +122,23 @@ export function activate(ctx: vscode.ExtensionContext): void {
 				}
 			);
 
-			ctx.subscriptions.push(c.start());
-
 			c.onReady().then(() => {
+				const capabilities = c.initializeResult && c.initializeResult.capabilities;
+				if (capabilities && !capabilities.completionProvider) {
+					ctx.subscriptions.push(vscode.languages.registerCompletionItemProvider(GO_MODE, new GoCompletionItemProvider(), '.', '\"'));
+					vscode.window.showInformationMessage('Your installed version of `go-langserver` is out of date and does not support code completion. Falling back to default behavior.');
+				}
+
 				if (!languageServerExperimentalFeatures['format'] ||
```

Comment:
Should this check for `=== true` then, too?

Reply:
Good point, I've pushed a commit to fix it